### PR TITLE
Add Swagger support

### PR DESCRIPTION
**Pull request by marcegl91**
## Summary
- Add Swagger support
## Trello Card
https://trello.com/c/1R9Yo9KH/40-swagger

### CODE REVIEW

File: src/main/java/wolox/training/config/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+
+    @Bean
+    public Docket productApi() {
```
Comment (leandrocostawolox) [inline src/main/java/wolox/training/config/SwaggerConfig.java]: Just a comment here @marcegl91 : Why is the name of this method `productApi`? 🤔 
It's usually called just `api`

**Review by leandrocostawolox — COMMENTED** [review]
Great! Just one little comment 👼

**Review by leandrocostawolox — APPROVED** [review]
Great!

File: src/main/java/wolox/training/config/SwaggerConfig.java (hunk 1 above)
Comment (marcegl91) [reply src/main/java/wolox/training/config/SwaggerConfig.java]: Good catch, I've just changed it! I made that mistake by copypasting from another site that piece of code :sweat_smile:

**Pull request merged**